app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str):
+    command_list = command.split(" ")
+    path_out_list = command_list[2].split("/")
+    path = ""
+    if len(path_out_list) > 1:
+        for i in range(len(path_out_list) - 1):
+            path += f"{path_out_list[i]}\\"
```

Comment:
use `os.path.join` to concatenate parts of the path. In that case, your app will be cross-platform and will use either `/` or `\` depending on OS